### DISCUSSION TITLE
Port/Interface reservations on devices aka status field on interfaces/ports

### DISCUSSION THREAD

**Discussion by empusas** [netbox-community/netbox#12495] [Ideas]
I think most companies with a larger network team have a problem that they may allocate ports/interfaces on a device for a project/customer and want to document this, so that they are not taken by someone else for a different purpose.
Problem I see often is that cabling to other devices cannot be planned as the devices are not there yet and not all details known.

One way to do this would be to use the "mark connected" and leave some information about who, when and for what the port was reserved. 
Same if an interface/port is defect. You can mark it as disabled and leave a comment in the description.

But would it be much nicer to have s proper status field, just like we have for IP addresses, devices etc?

**Comment by jeremystretch** [comment]
Both tags and custom fields are well-suited for this purpose.

**Reply by empusas** [reply]
Yes possible, from an automation point of view it would not matter if you have to query a CF or a status field. But from point of integration in UI(tables, objects etc) I think a status field would be nicer.
